docs/node-operators/getting-started.mdx

```diff
@@ -25,54 +25,61 @@ This documentation is for the **Mainnet** release.
 
-:::note
-
-Windows is not officially supported at this time. However, it may be possible to use the [Docker instructions](./#docker). If you are interested in receiving a grant to develop Mina for Windows, please email grants@o1labs.org or reach out in the [Mina Protocol Discord server](https://discord.gg/minaprotocol).
+**Processor**: Only AMD64 CPU architecture is supported.
 
-:::
+**Hardware**: Sending and receiving MINA does not require any special hardware. 
 
-**Hardware**: Sending and receiving mina does not require any special hardware, but running a block producer on the Mina network currently requires:
+Running a block producer on the Mina network requires at least:
 
-- at least a 8-core processor
-- at least 16GB of RAM
+- 8-core processor
+- 16 GB of RAM
 
-Note that if you plan on running a snark worker node at the same time as a block producer, you may need more RAM. GPUs aren't currently required, but may be required for node operators when the protocol is upgraded.
+More RAM is required to run a SNARK worker node at the same time as a block producer.
 
 **Network**: At least 1 Mbps connection
 
 **VM Instances**:
-O(1) Labs has tested running nodes on several cloud providers, and recommends the following instances for basic node operator needs. Keep in mind that custom requirements as well as different cost constraints may require a different instance type.
 
-- AWS - [c5.2xlarge](https://www.ec2instances.info/?filter=c5.2xl&region=us-west-2&cost_duration=daily&selected=c5.2xlarge)
-- GCP - [c2-standard-8](https://cloud.google.com/compute/docs/machine-types)
-- Azure - [Standard_F8s_v2](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-compute#fsv2-series-1)
-- Digital Ocean - [c-8-16gib](https://cloud.digitalocean.com/droplets/new?size=c-8-16gib)
+O(1) Labs has tested running nodes on several cloud providers. We recommend the following instances for basic node operator needs. Custom requirements and different cost constraints might require a different instance type.
 
-Check out our [guide](https://youtu.be/M9lE2lgyVEo) for how to configure your Mina node on Google Cloud.
+- AWS [c5.2xlarge](https://www.ec2instances.info/?filter=c5.2xl&region=us-west-2&cost_duration=daily&selected=c5.2xlarge)
+- GCP [c2-standard-8](https://cloud.google.com/compute/docs/machine-types)
+- Azure [Standard_F8s_v2](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes-compute#fsv2-series-1)
+- Digital Ocean [c-8-16gib](https://cloud.digitalocean.com/droplets/new?size=c-8-16gib)
+
+Check out the [Connecting to the Mina Testnet (Google Cloud)](https://youtu.be/M9lE2lgyVEo) video for steps to configure your Mina node on Google Cloud.
 
 ## Installation
 
-### Ubuntu / Debian
+To install the latest [Mainnet **Stable** Release 1.4.0](https://github.com/MinaProtocol/mina/releases/tag/1.4.0), use the installation steps for your environment.
```

Comment:
Use [descriptive link text](https://developers.google.com/style/link-text) that matches the page we send our readers to